### PR TITLE
User process tests and ExampleQueuer unit tests

### PR DESCRIPTION
**Pull request by m-cappi**
We created a series of tests using the guidelines, you can find them below.

Tests:
- [User - Process Tests](https://github.com/friggframework/create-frigg-app/pull/10/files#diff-5ca59ff364a4a3ffd9cfca937f5ee3a448b517f3af7971aab00acd2131fce58d)
- [Example Queuer - Unit Tests](https://github.com/friggframework/create-frigg-app/pull/10/files#diff-83ae54905702b3e8bf4d48d7721a1008f09185cbe6ad82b494aa3d246b89df4e)
- [Login Component - Unit Tests](https://github.com/friggframework/create-frigg-app/blob/0b70d82ff77a6d3ef19ccfb4d3bf4bb1057f5bc0/packages/cfa-template/template/frontend/src/__tests__/Login.test.js)

Improvements:

- Updated jest.config
- Updated babel.config

### CODE REVIEW

File: packages/cfa-template/template/frontend/src/__tests__/Login.test.js [hunk 1]
```diff
@@ -1,43 +1,81 @@
-import { render, screen, cleanup } from '@testing-library/react';
-import renderer from 'react-test-renderer';
-import Login from '../components/Login';
+import React from "react";
+import * as redux from "react-redux";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { ToastContainer } from "react-toastify";
+import nock from "nock";
```
Comment (seanspeaks) [inline packages/cfa-template/template/frontend/src/__tests__/Login.test.js:5]: Ah, hello nock

**Review by seanspeaks — COMMENTED** [review]
I don't really have any blockers for merging, since we'll fix quickly if there are errors based on the package versions mentioned in the comment threads. But, I do have a few questions scattered throughout my review.

Two higher level questions came to mind:
1. What's the difference between a user process test and a user unit test? Would the unit test for the user just be either a unit test for the user model or a unit test for the user route? (Vs the process test is a combination of both?)
2. Would router tests be solely unit tests, or process tests? Does it make sense to have both types for a single file? (Unit Tests to ensure router does its router things correctly and mocks responses, process tests to confirm that the routes actually create database records?

I think that's it for now, looking forward to building off this

File: packages/cfa-template/template/backend/package.json [hunk 2]
```diff
@@ -62,6 +59,7 @@
         "serverless-webpack": "^5.7.1",
         "sinon": "^10.0.0",
         "sinon-chai": "^3.6.0",
+        "supertest": "^6.2.4",
```
[the remark this answers is not in the document]
Comment (gregoriomartin) [reply packages/cfa-template/template/backend/package.json:61]: supertest is used in each process test, to start and run the backend.

> supertest: It is a high-level abstraction for testing HTTP while still allowing you to drop down to the lower-level API provided by superagent. We use it to run the backend as a web server during the process tests.

File: packages/cfa-template/template/backend/test/process-tests/user.process.test.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+describe('Users process tests', () => {
+    let credentials = { username: 'demo@lefthook.com', password: 'demo' };
+    let token;
+
+    beforeAll(() => {
+        mongoose.connect(process.env.MONGO_URI);
+    });
+
+    afterAll(() => {
+        mongoose.disconnect();
```
[the remark this answers is not in the document]
Comment (m-cappi) [reply packages/cfa-template/template/backend/test/process-tests/user.process.test.js:46]: The globalSetup and globalTeardown from @friggframework/test-environment are actually setting up a virtual MongoDB in memory, but you'd still need to handle mongoose connection/disconnection to it

File: packages/cfa-template/template/backend/test/process-tests/user.process.test.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            .post('/user/login')
+            .send(credentials)
+            .then((response) => {
+                token = response.body.token;
+                expect(response.statusCode).toEqual(201);
+                expect(response.body.token).toBeTruthy();
+            });
+    });
+
+    it('Lists user baseline integrations', async () => {
```
[the remark this answers is not in the document]
Comment (gregoriomartin) [reply packages/cfa-template/template/backend/test/process-tests/user.process.test.js:72]: You are correct; `user.process.test` only applies to the first two tests. The name will be modified to `auth-router.process.test`, which appears to be more precise.

**Pull request merged**